packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormInputField/TransferFormInputField.scss

```diff
@@ -1,26 +1,21 @@
 .wallets-transfer-form-input-field {
     display: flex;
     flex-direction: column;
-    color: var(--system-light-2-general-text, #333);
     padding-right: 0.8rem;
 
     &__input {
         all: unset;
-        height: 100%;
+        position: absolute;
+        inset: 0;
         color: var(--system-light-2-general-text, #333);
-        /* desktop/subtitle/Sub 2 - bold */
-        font-size: 2rem;
-        font-style: normal;
-        font-weight: 700;
-        line-height: 3rem; /* 150% */
-        width: 100%;
-    }
 
-    &__label {
-        /* desktop/paragraph/P2 - regular */
-        font-size: 1.4rem;
-        font-style: normal;
-        font-weight: 400;
-        line-height: 2rem; /* 142.857% */
+        &:disabled {
+            color: var(--system-light-4-disabled-text, #d6d6d6);
+        }
+
+        &-container {
```

Comment:
minor: if its block, then should be __container

Reply:
then probably better to keep it flat (as per bem recommendation, to make it more readable), so e.g. 
__input-container 
(still v. minor)

Reply:
This is done